src/components/home/elements/IssueToken.vue

```diff
@@ -0,0 +1,752 @@
+            scanShow: false,
+            sendError: false,
+            coldSignature: '',
+            timeStamp: Date.now() * 1e6,
+            hasConfirmed: false
+        }
+    },
+    props: {
+        tokenBalance: {
+            type: BigNumber,
+            default: function() {
+                return BigNumber(0)
+            },
+            require: true
+        },
+        balance: {
+            type: BigNumber,
+            default: function() {
+            },
+            require: true
+        },
+        tokenUnity: {
+            type: BigNumber,
+            default: function() {
+            },
+            require: true
+        },
+        coldAddresses: {
+            type: Object,
+            default: function() {},
+            require: true// Can not be removed ,cause this is an inherited property may be used somewhere
+        },
+        addresses: {
+            type: Object,
+            default: function() {},
+            require: true
+        },
+        maxSupply: {
+            type: BigNumber,
+            default: function() {
+            },
+            require: true
+        },
+        currentSupply: {
+            type: BigNumber,
+            default: function() {
+            },
+            require: true
+        },
+        walletType: {
+            type: String,
+            default: 'hotWallet',
+            require: true
+        },
+        tokenId: {
+            type: String,
+            default: '',
+            require: true
+        },
+        issuer: {
+            type: String,
+            default: '',
+            require: true
+        },
+        address: {
+            type: String,
+            default: ''
+        }
+    },
+    computed: {
+        contractId() {
+            return transaction.tokenIDToContractID(this.tokenId)
+        },
+        defaultAddress() {
+            return Vue.ls.get('address')
+        },
+        defaultColdAddress() {
+            if (this.noColdAddress) return ''
+            return Object.keys(this.coldAddresses)[0]
+        },
+        userInfo() {
+            return JSON.parse(window.localStorage.getItem(this.defaultAddress))
+        },
+        secretInfo() {
+            return JSON.parse(
+                seedLib.decryptSeedPhrase(this.userInfo.info, Vue.ls.get('pwd')))
+        },
+        seedPhrase() {
+            return seedLib.decryptSeedPhrase(this.secretInfo.encrSeed, Vue.ls.get('pwd'))
+        },
+        wordList() {
+            return this.seedPhrase.split(' ')
+        },
+        isSubmitDisabled() {
+            return !(BigNumber(this.amount).isGreaterThan(0) && this.isValidIssuer(this.address) && (this.isValidAttachment || !this.attachment) && this.isAmountValid('hot'))
+        },
+        isColdSubmitDisabled() {
+            return !(BigNumber(this.amount).isGreaterThan(0) && this.isValidIssuer(this.address) && (this.isValidColdAttachment || !this.coldAttachment) && this.isAmountValid('cold'))
+        },
```

Comment:
it seems that `isValidColdAttachment` and `coldAttachment` already removed.
amount only one version, so why `this.isAmountValid('cold')` with type 'cold'?
seems there is no difference between `isColdSubmitDisabled` and `isSubmitDisabled`.

Reply:
same in burn file